vellum.yaml

```diff
@@ -265,10 +283,10 @@ resources:
 
 outputs:
   public_mgmt_ip:
-    description: IP address in public management network
+    description: IP address in public (management) network
     value: { get_attr: [ mgmt_floating_ip, floating_ip_address ] }
   private_mgmt_ip:
-    description: IP address in private management network
+    description: IP address in private signaling network
```

Comment:
I think the changes here have got a bit confused. Don't think these entries needed changing